File: app/src/main/java/com/morozione/clicker/LevelsManager.java
package com.morozione.clicker;

/**
 * Created by morozione on 10/13/17.
 */

public class LevelsManager {
    private int level = 1;
    private int startRecord = 35;

    public void upperLevel() {
        level++;
        startRecord += level*20;
    }

    public int getStartRecord() {
        return startRecord;
    }

    public int getLevel() {
        return level;
    }
}

File: app/src/main/java/com/morozione/clicker/LevelsManager.kt
package com.morozione.clicker

import javax.inject.Inject

class LevelsManager @Inject constructor() {
    private var level: Int = 1
    private var startRecord: Int = 35

    fun upperLevel() {
        level++
        startRecord += level * 20
    }

    fun getStartRecord(): Int = startRecord

    fun getLevel(): Int = level
} 